app/src/main/java/org/oppia/app/model/Units.kt

```diff
@@ -0,0 +1,16 @@
+package org.oppia.app.model
+
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+
+class Units(
```

Comment:
This ought to be a data class, and since we aren't interacting with GSON I don't think we need the annotations:

```kotlin
data class Unit(val name: String, val aliases: List<String>, val frontUnits: List<String>, val baseUnit: String? = null)
```

Also note ``val``--data classes should be immutable.

Reply:
Done.